css/index.css

```diff
@@ -1,3 +1,362 @@
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+@media (max-width: 800px) {
+    .container{
+        max-width: 800px;
+    }
+}
+
+@media (max-width: 500px) {
+    .container{
+        max-width: 500px;
+    }
+}
+
+
+
+/* header */
+
+
+header nav {
+  display: inline-block;
+  width: 691px;
+  vertical-align: top;
+  margin-top: 35px;
+  
+}
+
+header nav a {
+  margin-right: 65px;
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  margin-top: 25px;
+}
+
+/* header media tablet -smaller than 800*/
+@media (max-width: 800px) {
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        align-items: center;
+    }
+
+    header nav {
+        display:flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-around;
+        /* margin-top:20px; */
+    }
+
+}
+
+
+
+/* header media phone -smaller than 500 */
+@media (max-width: 500px){
+    header nav {
+        display:flex;
+        flex-direction: column;
+        align-items: center;
+        margin-top:20px;
+        width: 100%;
+    }
+
+    header nav a{
+        width:100%;
+        text-align: center;
+        padding-top: 15px;
+        padding-bottom:15px;
+        margin-right:0px;
+        border-bottom: 1px solid grey;
+    }
+
+    header nav a:first-child {
+        border-top: 1px solid grey;
+    }
+}
+
+
+/* cta */
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+/* cta tablet 800 */
+@media (max-width: 800px){
+
+    .cta .cta-text{
+        display:flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-top: 0px;
+        padding-top: 60px; 
+    }
+   
+    .cta .cta-text button {
```

Comment:
now that we have explored other ways to make our layout responsive we can implement %'s, em and rems